### PR TITLE
Add link to events page in Weekly Online Events section

### LINKED ISSUE #1579: Weekly online events link on the Dashboard directs to forum root instead of relevant event post in the forum
Subject: forums link on the Dashboard
Description:
follow the forums-link on "Weekly Online Events"

Results:
link directs to the forum root. I think it would be better to direct to "community.couchers.org/c/23", that's the events category

Backend version: develop-0e8e0a37
Frontend version: develop-0e8e0a37
User Agent: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:89.0) Gecko/20100101 Firefox/89.0
Page: https://app.couchers.org/
User: nolo (14)

### PR DESCRIPTION
**Pull request by kerrylaval**
Instead of linking to forums page, links to events page directly
fixes #1579 

**Frontend checklist**
- [x] Formatted my code with `yarn format && yarn lint --fix`
- [x] There are no warnings from `yarn lint`
- [x] There are no console warnings when running the app
- [ ] Added any new components to storybook
- [ ] Added tests where relevant
- [x] All tests pass (except for tests with dates)
- [x] Clicked around my changes running locally and it works
- [ ] Checked Desktop, Mobile and Tablet screen sizes

### CODE REVIEW

**Review by aapeliv — DISMISSED** [review]
LGTM!

**Comment by kerrylaval** [comment]
@lucaslcode 


image

links to this page:

image

File: app/frontend/src/features/dashboard/constants.ts [hunk 1]
```diff
@@ -47,7 +47,7 @@ The events are held every week on:
 
 The events are held on Zoom at this link: [https://couchers.org/weekly-social](https://couchers.org/weekly-social).
 
-More information can be found on the [forums](https://community.couchers.org/)!`;
+More event information can be found [here](https://community.couchers.org/c/announcements/events/23)!`;
```
Comment (lucaslcode) [inline app/frontend/src/features/dashboard/constants.ts]:
```suggestion
You can find [more event information here](https://community.couchers.org/c/announcements/events/23)!`;
```

This way is more accessible - people that use screen readers often flick through links by themselves, so links should contain information about where they are going.

**Review by lucaslcode — CHANGES_REQUESTED** [review]
Thanks! Just a small nit

**Review by lucaslcode — APPROVED** [review]
Thanks for your contribution! :)

**Pull request merged**